src/SFA.DAS.RoatpGateway.Web/Services/GatewayApplicationActionsOrchestrator.cs

```diff
@@ -22,17 +22,20 @@ public async Task<RoatpWithdrawApplicationViewModel> GetWithdrawApplicationViewM
         {
             RoatpWithdrawApplicationViewModel viewModel = null;
 
-            var commonDetails = await _applyApiClient.GetPageCommonDetails(applicationId, _commonDetailsPageId, userName);
-            if (commonDetails != null)
```

Comment:
was this change from commonDetails intentional? Maybe @MarkFCain can advise as I thought it followed some sorta template

Reply:
It looks like this page needs stuff that isn't in common details (like referenceNumber, contact email), so these changes are fine imo.  You need to go get application and contact details anyway, so no need to do another call to common details here.